data/_ui/interfaces.txt

```diff
@@ -185,10 +185,10 @@ interface "map detail panel"
 interface "map planet card"
 	value "text start" 13
 	value "category size" 20
-	value "height" 130
+	value "categories" 5
+	value "extra height" 30
```

Comment:
Might "height padding" by a more descriptive name?

Reply:
Yeah that does sound better